test/compatibilityTestApi.spec.ts

```diff
@@ -1,50 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { AlfrescoApiConfig } from '../src/alfrescoApiConfig';
-import { AlfrescoApiCompatibility } from '../src/alfrescoApiCompatibility';
```

Comment:
Can this be removed? Intellij shows it is used only in md files: 
<img width="1512" alt="image" src="https://github.com/Alfresco/alfresco-js-api/assets/115619721/123def7e-1961-4f62-a74a-d551d322820b">